lib/remote_helper.py

```diff
@@ -313,8 +314,14 @@ def enable_root_login(cls, host, keyname, infrastructure, is_verbose):
       # logged-in user, so log in as that user to enable root login.
       if infrastructure == "gce":
         AppScaleLogger.log("Root login not enabled - enabling it now.")
-        cls.ssh(host, keyname, 'sudo cp ~/.ssh/authorized_keys /root/.ssh/',
+        temp_file = tempfile.NamedTemporaryFile()
```

Comment:
Not sure how to make it easy to run mktemp remotely and catch the name of the file: pipes and redirections are tricky when using sudo. The remote file was left behind: I will have a patch to remove it. Thanks for catching it!